.werft/util/werft.ts

```diff
@@ -107,6 +107,11 @@ export class Werft {
     }
 
     public endAllSpans() {
+        const traceID = this.rootSpan.spanContext().traceId
+        const nowUnix =  new Date().getTime() / 1000;
+        // At the moment we're just looking for traces in a 30 minutes timerange with the specific traceID
+        // A smarter approach would be to get a start timestamp from tracing.Initialize()
+        exec(`werft log result -d "Honeycomb trace" -c github-check-honeycomb-trace url https://ui.honeycomb.io/gitpod/datasets/werft/trace?trace_id=${traceID}&trace_start_ts=${nowUnix - 1800}&trace_end_ts=${nowUnix + 5}`);
```

Comment:
hey @mads-hartmann, let me borrow your typescript knowledge a little bit here

For some reason, this result is being cut after the traceID. `trace_start_ts` and `trace_end_ts` are not being added to the result...?

Because of that, the result does work for some minutes, but after 1~2 hours Honeycomb can't find it anymore. Honeycomb needs `trace_start_ts` and `trace_end_ts` for it to work without time restrictions

Reply:
Nevermind, just adding double quotes solved the issue :)